packages/@stylexjs/postcss-plugin/package.json

```diff
@@ -8,12 +8,28 @@
     "url": "git+https://github.com/facebook/stylex.git"
   },
   "license": "MIT",
+  "scripts": {
+    "test": "jest"
+  },
+  "jest": {
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "/__fixtures__/"
+    ],
+    "testEnvironment": "node"
+  },
   "dependencies": {
     "@babel/core": "^7.26.8",
     "@stylexjs/babel-plugin": "0.12.0",
     "postcss": "^8.4.49",
     "fast-glob": "^3.3.2",
     "glob-parent": "^6.0.2",
     "is-glob": "^4.0.3"
+  },
+  "devDependencies": {
+    "@babel/preset-env": "^7.26.0",
+    "@babel/preset-react": "^7.25.3",
+    "jest": "^30.0.0-alpha.7",
+    "react": "^18.2.0"
```

Comment:
This shouldn't be needed

Reply:
Fixed in https://github.com/facebook/stylex/pull/1067/commits/ce048a61aca0764a5f77b827e8517005d307e2a2